api/models/url.js

```diff
@@ -0,0 +1,32 @@
+const Sequelize = require("sequelize");
+const { createDbConnection } = require("../database/connection");
+
+const sequelize = createDbConnection();
+
+const getUrlTable = () => {
+  const tableName =
```

Comment:
we want to call a generic table name from the env vars. It will benefit us in the future when we want to add more deployment envs. Future proofing it.